src/ansys_sphinx_theme/theme/ansys_sphinx_theme/_templates/autoapi/python/class.rst

```diff
@@ -14,7 +14,38 @@
             - {{ obj.summary }}
           {% endfor %}
 {%- endmacro %}
-{# --------------------------- End macros definition ----------------------- #}
+
+{# ------------------------ End macros definition for tab ------------------- #}
+
+{# ----------------- Start macros definition for autosummary -----------------#}
+
+{% macro add_auto_summary_attribute(heading, id, types) -%}
```

Comment:
We are not using the `id` parameter in this macro:

```suggestion
{% macro add_auto_summary_attribute(heading, types) -%}
```

Reply:
good catch, it is not required